docs/source/index.rst

```diff
@@ -26,7 +26,7 @@ Four subsystems make up JupyterHub:
 * an **authentication class** that manages how users can access the system
 
 
-Besides these central pieces, you can add optional configurations through a `config.py` file and manage users kernels on an admin panel. A simplification of the whole system can be seen in the figure below:
+Besides these central pieces, you can add optional configurations through a `config.py` file and manage users kernels on an admin panel. A very detailed explination can be found in ":doc:`What is JupyteHub? <getting-started/what-is-jupyterhub>`". A simplification of the whole system can be seen in the figure below:
```

Comment:
```suggestion
Besides these central pieces, you can add optional configurations through a `config.py` file and manage users kernels on an admin panel. A very detailed explanation can be found in "[What is JupyteHub?](getting-started/what-is-jupyterhub)". A simplification of the whole system can be seen in the figure below:
```

Reply:
put the subsystems at the top so that users have that context as they're reading the docs